tests/test_millenium_patron.py

```diff
@@ -382,6 +392,27 @@ def test_patron_dump_to_patrondata_restriction_field(self):
         patrondata = api.patron_dump_to_patrondata('alice', content)
         eq_("10", patrondata.library_identifier)
 
+    def test_neighborhood(self):
+        # The value of PatronData.neighborhood depends on the 'neighborhood mode' setting.
+
+        # Default behavior is not to gather neighborhood information at all.
+        api = self.mock_api()
+        content = api.sample_data("dump.success.html")
+        patrondata = api.patron_dump_to_patrondata('alice', content)
```

Comment:
This is an example of what the ILS returns?

Reply:
Yep, it's a small HTML document with a bunch of lines separated by <BR> tags.